content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -16,22 +15,65 @@ options(
   install.packages.check.source = "no",
   install.packages.compile.from.source = "never"
 )
-if (!"checklist" %in% rownames(utils::installed.packages())) {
-  utils::install.packages("checklist")
+if (
+  !all(
+    c("checklist", "fortunes", "remotes") %in%
+    rownames(utils::installed.packages())
+  )
+) {
+  if (Sys.getenv("RSTUDIO") == "1") {
+    warning(
+      "Missing packages. ",
+      "Please close RStudio and run R-",
+      paste(R.Version()[c("major", "minor")], collapse = "."),
+      "-win.exe first to install them automatically."
+    )
+  } else {
+    # remove any existing LOCK file
+    file.remove(
+      list.files(
+        file.path(.libPaths(), "00LOCK"), recursive = TRUE,
+        full.names = TRUE
+      )
+    )
+    if (!"remotes" %in% rownames(utils::installed.packages())) {
+      message("Installing `remotes`, may take some time")
+      utils::install.packages(
+        "remotes", repos = "https://cloud.r-project.org/", dependencies = TRUE
+      )
+    }
+    stopifnot(
+      "Failed to install package `remotes`." =
+      "remotes" %in% rownames(utils::installed.packages())
+    )
+    if (!"checklist" %in% rownames(utils::installed.packages())) {
+      message("Installing `checklist`, may take some time")
+      remotes::install_cran(
+        "checklist", repos = "https://inbo.r-universe.dev", upgrade = "always",
+        dependencies = TRUE
+      )
+    }
+    if (!"fortunes" %in% rownames(utils::installed.packages())) {
+      message("Installing `fortunes`, may take some time")
+      remotes::install_cran(
+        "fortunes", repos = "https://cloud.r-project.org/", upgrade = "always",
+        dependencies = TRUE
```

Comment:
I suggest to begin `Rprofile.site` by setting:

```r
rrepos <- 
    c(CRAN = "https://cloud.r-project.org/",
      INLA = "https://inla.r-inla-download.org/R/stable",
      inbo = "https://inbo.r-universe.dev")
```

and then re-use as `options(repos = rrepos)` and above as `rrepos["inbo"]` etc. Avoiding hardcoding the URLs in multiple places.

Reply:
Fixed in 46ab938
I removed the hard coded URL as the default is `getOption("repos")`.